yunwoo/programmers-1/자연수-뒤집어-배열로-만들기.js

```diff
@@ -0,0 +1,7 @@
+function solution(n) {
+  return n
+    .toString()
+    .split("")
+    .map((v) => Number(v))
+    .reverse();
+}
```

Comment:
```suggestion
function solution(n) {
  return n
    .toString()
    .split("")
    .map(Number)
    .reverse();
}
```
- 잘푸셨네요 고생하셨습니다! map돌릴 때 콜백함수아니고 바로 `Number` 생성자 함수만 넘겨줘도 모든 요소가 숫자로 형변환 된다는 거 참고해주세요 👍

Reply:
헉 value값이 없이도 되는군요..! 처음알았습니다 감사합니다🙇‍♂️👍